src/components/InvoiceTimer/stories.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import React from 'react';
+
+import { storiesOf } from '@storybook/react/dist/client/preview';
+import { array, select, text, boolean, number } from '@storybook/addon-knobs';
+
+import InvoiceTimer from './InvoiceTimer';
+
+storiesOf('Invoice Timer', module)
+	.addDecorator((story) => (
+		<div style={{ display: 'inline-block', minWidth: 150 }}>{story()}</div>
+	))
+	.add(
+		'Default',
+		() => {
+			return <InvoiceTimer secondsRemaining={600} />;
```

Comment:
Add a knob on seconds remaining so it can be manipulated from storybook

Reply:
Still seems broken, not seeing a knob or an example in the storybook anymore
`NaN:NaN`